### PR TITLE
release 8.0.0

### LINKED ISSUE #415: Release bump
Can we please get a release to bring newer debian/centos version support (has already been patched, just not released)? Thanks

### PR DESCRIPTION
**Pull request by Dan33l**
#### This Pull Request (PR) fixes the following issues

Fixes #415

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+# Reference
```
Comment (alexjfisher) [inline REFERENCE.md]: I wouldn't generate this if no parameters have actually be documented using puppet-strings.

Comment (Dan33l) [reply REFERENCE.md]: So, shouldn't rake task take care of this ?

Comment (alexjfisher) [reply REFERENCE.md]: which task did you run?

Comment (Dan33l) [reply REFERENCE.md]: this `bundle exec rake strings:generate:reference`

Comment (alexjfisher) [reply REFERENCE.md]: oh right.  yeah, that doesn't have any concept of whether the result are going to be worth it.

Comment (Dan33l) [reply REFERENCE.md]: The file is now removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v8.0.0](https://github.com/voxpupuli/puppet-nodejs/tree/v8.0.0) (2020-03-05)
```
Comment (alexjfisher) [inline CHANGELOG.md]: bump the date forward before merging, but otherwise +1

**Pull request merged**